pkg/storage/snapshot/snapshot_test.go

```diff
@@ -1151,12 +1147,11 @@ var _ = Describe("Snapshot controlleer", func() {
 
 				updatedSnapshot := vmSnapshot.DeepCopy()
 				updatedSnapshot.Status.VirtualMachineSnapshotContentName = &vmSnapshotContent.Name
-				updatedSnapshot.Status.CreationTime = timeFunc()
 				updatedSnapshot.Status.ReadyToUse = &f
 				updatedSnapshot.Status.Indications = nil
 				updatedSnapshot.Status.Conditions = []snapshotv1.Condition{
 					newProgressingCondition(corev1.ConditionFalse, "In error state"),
-					newReadyCondition(corev1.ConditionFalse, "Error"),
+					newReadyCondition(corev1.ConditionFalse, "Not ready"),
```

Comment:
Connects to my readycondition reason comment, I would prefer here that it will stay Error. I believe the UI are now presenting this condition in the vmsnapshot tab so it will be better to have the reason of `Error` to why the vmsnapshot is not ready